keras_cv/layers/preprocessing/rand_augment.py

```diff
@@ -0,0 +1,277 @@
+            self.translate_y,
+            self.cutout,
+        ]
+        self.auto_vectorize = False
+
+    def _augment(self, sample):
+        sample["images"] = preprocessing_utils.transform_value_range(
+            sample["images"], self.value_range, (0, 255)
+        )
+        augmented_sample = sample
+        for _ in range(self.num_layers):
+            selected_op = tf.random.uniform(
+                (), maxval=len(self.augmentation_layers) + 1, dtype=tf.int32
+            )
+            branch_fns = []
+            for (i, layer) in enumerate(self.augmentation_layers):
+                branch_fns.append((i, lambda: layer(sample)))
+
+            augmented_sample = tf.switch_case(
+                branch_index=selected_op, branch_fns=branch_fns, default=lambda: sample
+            )
+            if self.probability_to_apply is not None:
+                augmented_sample = tf.cond(
+                    tf.random.uniform(shape=(), dtype=tf.float32)
+                    < self.probability_to_apply,
+                    lambda: augmented_sample,
+                    lambda: sample,
+                )
+
+        augmented_sample["images"] = preprocessing_utils.transform_value_range(
+            augmented_sample["images"], (0, 255), self.value_range
+        )
+        return augmented_sample
+
+
+def auto_contrast_policy(magnitude, magnitude_std):
+    return {}
+
+
+def equalize_policy(magnitude, magnitude_std):
+    return {}
+
+
+def solarize_policy(magnitude, magnitude_std):
+    threshold_factor = core.NormalFactorSampler(
+        mean=(magnitude / 10) * 256,
+        standard_deviation=(magnitude_std / 10.0) * 256,
+        min_value=0,
+        max_value=255,
+    )
+    return {"threshold_factor": threshold_factor}
+
+
+def solarize_add_policy(magnitude, magnitude_std):
+    addition_factor = core.NormalFactorSampler(
+        mean=(magnitude / 10) * 110,
+        standard_deviation=(magnitude_std / 10.0) * 110,
+        min_value=0,
+        max_value=110,
+    )
+    return {"addition_factor": addition_factor, "threshold_factor": 128}
+
+
+def invert_policy(magnitude, magnitude_std):
+    return {"addition_factor": 0, "threshold_factor": 0}
+
+
+def color_policy(magnitude, magnitude_std):
+    factor = core.NormalFactorSampler(
+        mean=magnitude / 10.0,
+        standard_deviation=magnitude_std / 10.0,
+        min_value=0,
+        max_value=1,
+    )
+    return {"factor": factor}
+
+
+def contrast_policy(magnitude, magnitude_std):
+    # TODO(lukewood): should we integrate RandomContrast with `factor`?
+    # RandomContrast layer errors when factor=0
+    factor = max(magnitude / 10, 0.001)
+    return {"factor": factor}
+
+
+def brightness_policy(magnitude, magnitude_std):
+    # TODO(lukewood): should we integrate RandomBrightness with `factor`?
+    return {"factor": magnitude / 10.0}
+
+
+def shear_x_policy(magnitude, magnitude_std):
+    factor = core.NormalFactorSampler(
+        mean=magnitude / 10.0,
```

Comment:
Since there are too many 10.0 here (which I assume is the max for magnitude), lets extract it into a constant.